src/pypromice/ressources/variables.csv

```diff
@@ -0,0 +1,91 @@
+field,standard_name,long_name,units,coverage_content_type,coordinates,instantaneous_hourly,where_to_find,lo,hi,OOL,station_type,L0,L2,L3,max_decimals
```

Comment:
Git isn't track this as an update to the original variables.csv because the original file is deleted in an older commit.
Consider updating the efbe35f885cdf3c894a480998b4e0f4274cb4822 to move variables.csv instead.

Reply:
The old and new variable.csv can now be compared in this commit: https://github.com/GEUS-Glaciology-and-Climate/pypromice/commit/352774fb402eecb47161039050f2a3beeab2990c